README.md

```diff
@@ -15,7 +29,7 @@ early state there are use cases for which it offers significant help.
 
 # Storing downloads locally
 
-For testing, make a new scratch directory, say `/tmp/repo/`, to specify
+Make a new scratch directory, say `/tmp/repo/`, to specify
```

Comment:
Might want to change the "scratch directory" part too then. It won't duplicate downloads so it could often make sense to locally accumulate downloads over time rather than use a temporary/scratch directory for them, it just downloads what's not already local.